app/helpers/admin_form_builder.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class AdminFormBuilder < ActionView::Helpers::FormBuilder
+  include ActionView::Helpers::FormTagHelper
+
+  attr_accessor :output_buffer
+
+  def field(attribute, options = {}, &block)
+    content = @template.capture(&block) if block_given?
+    options[:class] = options.key?(:class) ? Array(options[:class]) : []
+    options[:class] << 'field'
+    options[:class] << 'invalid' if @object.errors.include?(attribute)
```

Comment:
@ofsound In case you are wondering where the form classes in the admin are coming from...